### ISSUE TITLE
Email: IBM Comments on WCAG 3.0 Draft (4 - Scoring and Conformance)

### ISSUE THREAD

**Issue by cwadamsoforacle** [w3c/silver#498]
Scoring and Conformance
IBM applauds the working group’s intent of moving away from a "no-defects" conformance model to have conformance better able to reflect real-world accessibility, including making allowances for bugs and other low-impact issues. However, we think there is still work to do in this area. 
•	One way of adding granularity into the scoring and conformance reporting system could be to provide reporting by functional category. Combined with scoring by process, this would allow for assessors to judge sites by the priorities of the procurer/user and be scoped to particular user roles.
•	Question from Status section: How well will this proposal of conformance work in your type of organization and why? 

Response: It is really difficult to assess that. First, it is a little difficult to fully understand how the scoring and conformance will work in practice. Our current tools, test methods, and conformance reporting would have to change significantly to support the data needed to accurately calculate conformance using this model. Changes needed in tools: 
o	Not all test tools/test methods currently count all instances of elements tested or aggregate into a percentage score. 
o	Test rules would have to change for the requirements changes (e.g. text contrast) and the changes may require different splits between automated/manual testing.
o	Test rules would have to change from reporting pass/fail on the previous WCAG 2.x success criterion (often by Level A, AA & AAA) to break down to the specific WCAG 3 outcomes. 
o	The flattening of levels within WCAG to a single level means a significant number of new test rules would need to be developed. There are a great number of Level AAA criteria (28 in WCAG 2.1, more than 1/3 of the total number of WCAG criteria) that have no test cases, automated testing, or conformance reporting in the tools. If Silver also adds a large number of brand-new requirements (e.g., Clear words), then the cost of implementing all of these criteria could increase the cost and time of accessibility implementation by a factor of 35% or more. Having this large number of additional criteria would reduce the number of products and websites that can meet bronze level and may dissuade authors from trying.
o	Depending on the number of outcomes that require manual testing, the ability to count instances (manually) of applicable content, and appropriately test all of the outcomes could cost significant time.
o	Aggregating test results for very large, complex applications that have multiple components, multiple user roles, and multiple technologies and use the scoring methodologies could be a challenge, at least initially. Remember, the end-game for a product is to report the accessibility to potential customers and give enough granularity to enable meaningful comparisons between competing products. 
•	Does the model for scoring and aggregated ratings work? Why or why not? If not, please propose an alternative solution. 

Response: The current proposal is a bit difficult to understand exactly how it works without a good example. It would help to show how this could work in real-world situations, for both large-scale and small-scale application of the scoring methodology. Suggest that there be an explainer document developed that contains the WCAG 3 document structure and other background (removing that material from the standard), with an example of how the scoring methodology could be used.
•	The scoring and conformance are quite different from WCAG 2.x to WCAG 3. This means that persons who are trying to procure accessible products may find it difficult to compare products working to meet WCAG 2.x vs. WCAG 3. I know this is outside of the standard’s purview but must be considered when thinking of the roadmap to adoption.

**Comment by cwadamsoforacle** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by cwadamsoforacle** [comment]
[IBM comments on the WCAG 3 FPWD -March 2.docx](https://github.com/w3c/silver/files/6174023/IBM.comments.on.the.WCAG.3.FPWD.-March.2.docx)
Forgot to add attachment to original post.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
